homeassistant/components/nexia/scene.py

```diff
@@ -52,5 +52,5 @@ def icon(self):
         """Activate an automation scene."""
         await self.hass.async_add_executor_job(self._automation.activate)
         async_call_later(
-            self.hass, SCENE_ACTIVATION_TIME, self._coordinator.async_refresh
+            self.hass, SCENE_ACTIVATION_TIME, self._coordinator.async_refresh()
         )
```

Comment:
the callback of `async_call_later` is passed the time it is being called. By passing a `coro` you bypass this, but now if Home Assistant were to quit before this was called, it will print a warning for having an unawaited coro. 

It's cleaner if you create a function to handle it.

```suggestion
        async def refresh_callback(_):
            await self._coordinator.async_refresh()
        
        async_call_later(
            self.hass, SCENE_ACTIVATION_TIME, refresh_callback
        )
```

Reply:
Thanks for the detailed explanation. Very helpful. All fixed up